dnsimple/zones_records_test.go

```diff
@@ -90,28 +93,37 @@ func TestZonesService_CreateRecord(t *testing.T) {
 		testMethod(t, r, "POST")
 		testHeaders(t, r)
 
-		want := map[string]interface{}{"name": "foo", "content": "192.168.0.10", "type": "A"}
-		testRequestJSON(t, r, want)
+		// --- FAIL: TestZonesService_CreateRecord (0.00s)
```

Comment:
@weppos I [declared](https://github.com/dnsimple/dnsimple-go/pull/47/files#diff-3df835be8edaab5a6734faa367081eeaR13) `Region.Name` as `omitempty`, but it still forced as key in the request payload for create/update.

I commented out this assertion because there is something magical ✨  happening here. Can you please check? Thanks.